### PR TITLE
Auto update packages

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

**Comment by icemac** [comment]
I have to merge this immediately because the TravisCI import feature does not seem to support to define a branch to look for the base file for import.

**Comment by dataflake** [comment]
I have a few more suggestions for `.gitignore`:

These are created when you set up your buildout with `python3 -mvenv` and run builds:

- pyvenv.cfg
- /lib64

These seem to be created on macOS every time I build something with a virtual environment:

- *.profraw

**Comment by icemac** [comment]
See https://github.com/zopefoundation/zLOG/pull/4 for an example PR created by the script.

File: config/pure-python-without-pypy/tox.ini [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+[testenv:coverage]
+basepython = python3
+deps =
+    coverage
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report
+
+[coverage:run]
```
Comment (dataflake) [inline config/pure-python-without-pypy/tox.ini]: Does this coverage configuration get picked up? I would expect that to be in setup.cfg. If it gets read from here that's fine, I am just wondering.

Comment (icemac) [reply config/pure-python-without-pypy/tox.ini]: Yes, it should according to https://coverage.readthedocs.io/en/v4.5.x/config.html#configuration-files (last paragraph of first section).

File: config/pure-python-without-pypy/gitignore [hunk 2]
```diff
@@ -0,0 +1,14 @@
+*.egg-info
+*.pyc
+*.pyo
```
Comment (mgedmin) [inline config/pure-python-without-pypy/gitignore]: I suggest also adding `__pycache__/`, although the above rules would probably pick everything under it anyway.

File: config/pure-python-without-pypy/gitignore [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+.installed.cfg
+.mr.developer.cfg
+.tox/
+bin/
+coverage.xml
+develop-eggs/
+dist/
+htmlcov/
+lib/
+parts/
```
Comment (mgedmin) [inline config/pure-python-without-pypy/gitignore]: What about `build/`?  And `eggs/`?

File: config/pure-python-without-pypy/gitignore (hunk 2 above)
Comment (icemac) [reply config/pure-python-without-pypy/gitignore]: I think `__pycache__` is Python 3 only.

File: config/pure-python-without-pypy/gitignore (hunk 3 above)
Comment (icemac) [reply config/pure-python-without-pypy/gitignore]: I'm going to remove `build/` as I do not know where it is needed.
`eggs/` is the default place where eggs are stored by buildout if not using a global eggs directory.

Comment (icemac) [reply config/pure-python-without-pypy/gitignore]: Sorry, not read correctly. I am going to add `eggs/` but what is `build/` for?

Comment (iham) [reply config/pure-python-without-pypy/gitignore]: build is created when using zestreleaser or such

Comment (dataflake) [reply config/pure-python-without-pypy/gitignore]: I think it's created when you build an egg in some situations, like a binary compile?

Comment (dataflake) [reply config/pure-python-without-pypy/gitignore]: It seems to be created when you run bdist_wheel

Comment (iham) [reply config/pure-python-without-pypy/gitignore]: most importantly: you can add build/ to gitignores ;) you'll never want it to be added or checked in

Comment (icemac) [reply config/pure-python-without-pypy/gitignore]: Okay, I am going to add `build/`, too.

**Pull request merged**